### DISCUSSION TITLE
ConnectorError "failed to lookup address information: Name or service not known"

### DISCUSSION THREAD

**Discussion by valerebron** [prisma/prisma#3363] [Q&A]
I'm running prisma in a nodejs app with apollo server, vue apollo for client side and all query/mutation works perfectly.
Besides that I need to directly trigger a query with nodejs (so on the server side) without passing by apollo API.

I try several ways to do that, but each time I call the resolver direclty in my nodejs script,
this error appended :

`Error occurred during query execution:`
`ConnectorError "failed to lookup address information: Name or service not known"`

Here is my script : 

```
import { ApolloServer } from 'apollo-server'
import { PrismaClient } from '@prisma/client'

const prisma = new PrismaClient()
const api = new ApolloServer()

// apollo API works great :)
api.listen()

// direct call, connector error: name or service not known :(
let main = async function() {
    let users = await prisma.user.findMany()
}

main()
```

I don't understand why apollo server can't connect to prisma client but nodejs can't ?

**Comment by ryands17** [comment]
Hey @valerebron 👋 
I don't understand exaclty what you're trying to achieve and where is the `Query` coming from?

**Comment by valerebron** [comment]
Hi ryands, I'l try to be more explicit.
My app need a 'maintenance function' (like a cron in linux) to automated som tasks.
To do that I need to trigger a mutation **from the server side** and not from the client side.
Nevermind the `Query`, I've just simplify my code up there :)

**Reply by ryands17** [reply]
It would be great if you could share your code. Also I don't think the best way to call a query would be the function itself directly, but use something like [graphql-request](https://github.com/prisma-labs/graphql-request/) to make your query, passing the Apollo server endpoint.
